miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniappSdkInitializer.kt

```diff
@@ -72,7 +78,7 @@ class MiniappSdkInitializer : ContentProvider() {
                 subscriptionKey = manifestConfig.subscriptionKey(),
                 hostAppVersionId = manifestConfig.hostAppVersion(),
                 hostAppUserAgentInfo = manifestConfig.hostAppUserAgentInfo(),
-                isTestMode = manifestConfig.isTestMode()
+                isTestMode = manifestConfig.isPreviewMode()
```

Comment:
`isTestMode` is deprecated, but it should still work if they have only set that value and not `isPreviewMode`. So maybe this instead?
```suggestion
                isTestMode = manifestConfig.isPreviewMode() || manifestConfig.isTestMode()
```

Reply:
I have updated this part.